packages/design-system/src/components/VerticalNav/VerticalNavItemLabel.jsx

```diff
@@ -64,7 +65,12 @@ export class VerticalNavItemLabel extends React.PureComponent {
       props = Object.assign(props, this.anchorProps());
     }
 
-    return <this.LabelComponent {...props}>{this.props.label}</this.LabelComponent>;
+    return (
+      <this.LabelComponent {...props}>
+        {this.props.label}
+        {this.props.hasSubnav && <ArrowIcon direction={this.props.collapsed ? 'down' : 'up'} />}
```

Comment:
For this one, the weight feels different

![vertical-nav](https://user-images.githubusercontent.com/7595652/147167145-fc26cdff-ce3c-45bd-8a02-c1b3088b6d99.gif)

Reply:
The weight here is different. As part of consolidating all the arrow icons in the design system, we had to go with a lighter-weight icon.